module/zfs/dmu_zfetch.c

```diff
@@ -196,6 +196,19 @@ dmu_zfetch_stream_create(zfetch_t *zf, uint64_t blkid)
 	list_insert_head(&zf->zf_stream, zs);
 }
 
+int dmu_zfetch_out_of_range = 0;
+
```

Comment:
I'd recommend renaming this function and adding a comment, like:
```
/* Temporary debug print function to run down a case where max_dist is negative */
debug_max_dist_is_negative(char *constraint, uint64_t blkid, uint64_t nblks,
```

Reply:
Good idea.  Done.